src/api/status/public/js/pages/build.js

```diff
@@ -3,5 +3,5 @@ import './main.js';
 
 window.addEventListener('load', () => {
   checkForBuild();
-  setInterval(checkForBuild, 5000);
+  setInterval(checkForBuild, 30000);
```

Comment:
This can probably be every 10s, since you'll ignore it if the build hasn't changed.